input/pagecontent/medicine-information.md

```diff
@@ -26,12 +26,18 @@ Australian Medicines Terminology (AMT) is the national terminology for identific
 The AMT is published monthly to include new items on the Australian Register of Therapeutic Goods from the TGA, as well as items listed on the Pharmaceutical Benefits Scheme. 
 The AMT is published as part of SNOMED CT-AU (Australian edition of SNOMED CT) and can be downloaded in a variety of formats from the [National Clinical Terminology Service (NCTS)](https://www.healthterminologies.gov.au/).
 
-In addition to the medication code, the majority of use cases support exchange of structured medicine information as separate data elements covering brand name, generic name, item form and strength, and manufacturer.
+In addition to the medication code, the majority of use cases support exchange of structured medicine information as separate data elements covering brand name, generic name, item form and strength, and manufacturer. These data elements may be supported as coded, or text, and systems are likely to use a combination of coded and text elements when constructing a Medication resource. 
 
-These data elements may be supported as coded, or text, and systems are likely to use a combination of coded and text elements when constructing a Medication resource. The guidance for how to support coded or text is summarised below: 
+The guidance for how to support coded or text identification of medicinal products is summarised below: 
 
-1.  For *coded* support of a medication, a single code can be supplied in Medication.code if the code contains the information required.
-     
+1.  For *coded* support of a medication, information required for identification of a medication (i.e. brand name, generic name, form and strength) can be provided as a single code using code.coding, if the code contains that information. 
```

Comment:
point 1 should be worded as preferred. Make reference to the fact that depending on the terminology (noting AMT is preferred) that look up is sufficient to interrogate strength etc. substances, ingredients, etc.

Reply:
Done - hopefully not too detailed now